src/components/visx/TimeSeriesChart.tsx

```diff
@@ -155,7 +155,9 @@ export const TimeSeriesChart = <Datum extends {}>({
   const [zoomDomain, setZoomDomain] = useState<number | undefined>(
     defaultZoomDomain
       ? getClampedZoomDomain(defaultZoomDomain)
-      : xAccessor(latestDatum) - xAccessor(earliestDatum)
+      : latestDatum != null && earliestDatum != null
+        ? xAccessor(latestDatum) - xAccessor(earliestDatum)
+        : minZoomDomain
```

Comment:
[suggestion] maybe i'm just bad at reading ternaries but would prefer this be a function. reading nested ternaries is a little annoying for me and we've run into issues with these getting messed up in the past